libtransmission/announcer.c

```diff
@@ -153,6 +181,32 @@ typedef struct tr_announcer
 }
 tr_announcer;
 
+static struct tr_scrape_info* tr_announcerGetScrapeInfo(struct tr_announcer* announcer, const char* url)
+{
+    struct tr_scrape_info* info = NULL;
+
+    if (url && *url)
```

Comment:
More to the point, you could short-circuit this to return `NULL` right away and thus decrease further code indent one level.

Reply:
I generally try to minimize the number of return points in C code since so much cleanup has to done manually. However there's no such overhead in this function and I'll change it if you feel strongly about it.